services/venice-controller/src/main/java/com/linkedin/venice/controller/systemstore/SystemStoreRepairTask.java

```diff
@@ -351,14 +427,23 @@ AtomicLong getBadPushStatusStoreCount(String clusterName) {
     return getClusterSystemStoreHealthCheckStats(clusterName).getBadPushStatusSystemStoreCounter();
   }
 
-  AtomicLong getUnreachableSystemStoreCounter(String clusterName) {
-    return getClusterSystemStoreHealthCheckStats(clusterName).getUnreachableSystemStoreCounter();
-  }
-
   AtomicLong getNotRepairableSystemStoreCounter(String clusterName) {
     return getClusterSystemStoreHealthCheckStats(clusterName).getNotRepairableSystemStoreCounter();
   }
 
+  boolean shouldSkipStoreCheck(String clusterName, Store store) {
+    if (!shouldContinue(clusterName)) {
```

Comment:
can we absorb the `shouldContinue` in this method and use it everywhere instead of just `shouldContinue` ?